src/scripts/main.js

```diff
@@ -1,3 +1,102 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const rowRemove = document.querySelector('.remove-row');
+const rowAdd = document.querySelector('.append-row');
+const columnAdd = document.querySelector('.append-column');
+const columnRemove = document.querySelector('.remove-column');
+
+document.addEventListener('click', (e) => {
+  const target = e.target.closest('.button');
+
+  if (!target) {
+    return;
+  }
+
+  changeTable(target);
+});
+
+function changeTable(elem) {
+  const addRow = elem.closest('.append-row');
+  const addColumn = elem.closest('.append-column');
+  const removeRow = elem.closest('.remove-row');
+  const removeColumn = elem.closest('.remove-column');
+
+  switch (elem) {
+    case addRow:
+      const cloneElem = table.rows[0].cloneNode(true);
+
+      table.prepend(cloneElem);
```

Comment:
The 'prepend' method is used here, which adds the new row to the beginning of the table. If the intention is to add the row to the end, consider using 'appendChild' instead.